sirepo/sim_data/flash.py

```diff
@@ -13,7 +13,7 @@
 
 class SimData(sirepo.sim_data.SimDataBase):
 
-    FLASH_RPM_FILENAME = 'flash.rpm'
+    FLASH_RPM = 'flash.rpm'
     _FLASH_EXE_PREFIX = 'flash4'
```

Comment:
`_FLASH_PREFIX` since it has two uses.

Reply:
https://github.com/radiasoft/sirepo/pull/2490/commits/cd07af6a29565ca8f4b5a3144ff997244442b7ea